hack/common.sh

```diff
@@ -248,6 +256,13 @@ os::build::place_bins() {
         find "${full_binpath_src}" -maxdepth 1 -type f -exec \
           rsync -pt {} "${OS_OUTPUT_BINPATH}/${platform}" \;
 
+        if [[ -f "${OS_OUTPUT_BINPATH}/${platform}/openshift" ]]; then
+          echo "++ Creating symlinks"
+          for linkname in "${OPENSHIFT_BINARY_SYMLINKS[@]}"; do
+            ln -s "${OS_OUTPUT_BINPATH}/${platform}/openshift" "${OS_OUTPUT_BINPATH}/${platform}/${linkname}"
```

Comment:
@smarterclayton It will:

    $ ls -l _output/local/bin/linux/amd64/
    total 38740
    -rwxr-xr-x. 1 pmorie 39643392 Jan 19 17:55 openshift
    lrwxrwxrwx. 1 pmorie       64 Jan 19 18:10 openshift-deploy -> /home/pmorie/code/origin/_output/local/bin/linux/amd64/openshift
    lrwxrwxrwx. 1 pmorie       64 Jan 19 18:10 openshift-docker-build -> /home/pmorie/code/origin/_output/local/bin/linux/amd64/openshift
    lrwxrwxrwx. 1 pmorie       64 Jan 19 18:10 openshift-experimental -> /home/pmorie/code/origin/_output/local/bin/linux/amd64/openshift
    lrwxrwxrwx. 1 pmorie       64 Jan 19 18:10 openshift-router -> /home/pmorie/code/origin/_output/local/bin/linux/amd64/openshift
    lrwxrwxrwx. 1 pmorie       64 Jan 19 18:10 openshift-sti-build -> /home/pmorie/code/origin/_output/local/bin/linux/amd64/openshift
    lrwxrwxrwx. 1 pmorie       64 Jan 19 18:10 osc -> /home/pmorie/code/origin/_output/local/bin/linux/amd64/openshift